Need to fix the errors detected with cargo clippy test

Command:
```
cargo clippy --target=x86_64-unknown-none --all-features
```

Error Log:
```
error: cannot find macro `asm` in this scope
  --> td-payload/src/bin/example/stack.rs:26:9
   |
26 |         asm!("mov {}, rsp", out(reg) rsp);
   |         ^^^
   |
help: consider importing this macro
   |
16 + use core::arch::asm;
   |

error[E0412]: cannot find type `RuntimeMemoryLayout` in this scope
  --> td-payload/src/bin/example/stack.rs:32:35
   |
32 | pub fn bench_stack(memory_layout: RuntimeMemoryLayout) {
   |                                   ^^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `memory_layout` in this scope
  --> td-payload/src/bin/example/main.rs:52:28
   |
52 |         stack::bench_stack(memory_layout);
   |                            ^^^^^^^^^^^^^ not found in this scope

Some errors have detailed explanations: E0412, E0425.
For more information about an error, try `rustc --explain E0412`.
error: could not compile `td-payload` (bin "example") due to 3 previous errors
```

--- a/td-payload/src/bin/example/main.rs
+++ b/td-payload/src/bin/example/main.rs
@@ -49,7 +49,7 @@ pub extern "C" fn main() -> ! {
 
     #[cfg(feature = "benches")]
     {
-        stack::bench_stack(memory_layout);
+        stack::bench_stack();
     }
 
     #[cfg(feature = "tdx")]

--- a/td-payload/src/bin/example/stack.rs
+++ b/td-payload/src/bin/example/stack.rs
@@ -13,6 +13,7 @@
 // limitations under the License.
 // This function will cause page fault by memory protection.
 
+use core::arch::asm;
 use td_benchmark::StackProfiling;
 use td_payload::{mm::layout::DEFAULT_STACK_SIZE, println};
 
@@ -29,7 +30,7 @@ fn test_stack() {
     let b = vec![1u8, 2, 3, 4];
 }
 
-pub fn bench_stack(memory_layout: RuntimeMemoryLayout) {
+pub fn bench_stack() {
     StackProfiling::init(0x5a5a_5a5a_5a5a_5a5a, 0x20_0000);
     test_stack();
     let stack_usage = StackProfiling::stack_usage().unwrap();

